deployment/roles/deploy/tasks/deploy-configmaps.yml

```
@@ -2,14 +2,15 @@
 # SPDX-License-Identifier: MIT
 
 ---
-- name: Deploy ConfigMap
+- name: Deploy ConfigMaps
   k8s:
     namespace: "{{ project }}"
     resource_definition: "{{ item }}"
     host: "{{ host }}"
     api_key: "{{ api_key }}"
     validate_certs: "{{ validate_certs }}"
   with_template:
-    - configmap.yml.j2
+    - cm-env-config.yml.j2
+    - cm-nginx.yml.j2
```

```suggestion
    - env-config-cm.yml.j2
    - nginx-cm.yml.j2
```

Just to have a consistent naming across the board.